tests/test_plugin.py

```diff
@@ -7,17 +7,22 @@ def test_configure(mocker):
     config = mocker.Mock(
         cache={pytest_ruff.HISTKEY: mocker.sentinel.cache},
         option=mocker.Mock(ruff=True),
+        stash=pytest.Stash(),
     )
     pytest_ruff.pytest_configure(config)
-    assert config._ruffmtimes == mocker.sentinel.cache
+    assert config.stash[pytest_ruff._mtimes_stash_key] == mocker.sentinel.cache
 
 
 def test_configure_without_ruff(mocker):
     config = mocker.Mock(
         option=mocker.Mock(ruff=False),
+        stash=pytest.Stash(),
+        # Mocking to `not hasattr(config, "cache")`.
+        spec=["addinivalue_line", "option", "stash"],
     )
     pytest_ruff.pytest_configure(config)
-    assert hasattr(config, "_ruffmtimes")
```

Comment:
This assert was wrong because the mock was wrong. The `hasattr(config, "cache")` was `True` for mock config.